src/components/UpdateProfile.vue

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
This and RegisterView.vue share a lot of similarities, at one point we will refactor.